src/webconnector-tellus/src/scraper-cursor.js

```diff
@@ -4,7 +4,7 @@
 // tiny bit of data
 const MAX_PAGES = 1000000;
```

Comment:
This can be removed now that it is set by the user

Reply:
It's only set (indirectly) if the user selected a limit, otherwise some max number of pages seems wise for an possibly enormous dataset